tests/test_connection.py

```diff
@@ -111,6 +111,34 @@ def test_reconnect_channel_peers(node_factory, executor):
     fut3.result(10)
 
 
+def test_connection_moved(node_factory, executor):
+    slow_start = os.path.join(os.getcwd(), 'tests/plugins/slow_start.py')
+    options = {'may_reconnect': True, 'plugin': slow_start}
+    l1, l2 = node_factory.get_nodes(2, opts=options)
+
+    # This is the port the plugin listens on
+    hang_port = 6669
+
+    # Set up the plugin to wait for a connection
+    waitconn = executor.submit(l1.rpc.waitconn)
+    l1.daemon.wait_for_log('listening for connections')
+
+    # Attempt connection
+    fut_hang = executor.submit(l1.rpc.connect, l2.info['id'],
+                               'localhost', hang_port)
+
+    # Provide correct connection details
+    fut_success = executor.submit(l1.rpc.connect, l2.info['id'],
+                                  'localhost', l2.port)
```

Comment:
This can be just a normal call, since we only interleave with the `wait_for_log` which doesn't change its offset during RPC calls.

Reply:
Ah good point. Updated.